src/bundle.js

```diff
@@ -1,182 +0,0 @@
-/**
- * require.js module definitions bundled by webpack
- */
-// Use define from require.js not webpack's define
-const _define = window.define;
-
-// fetch
-const fetch = require('whatwg-fetch');
-_define('fetch', function() {
-    return fetch;
-});
-
-// Blurhash
-const blurhash = require('blurhash');
-_define('blurhash', function() {
-    return blurhash;
-});
-
-// query-string
-const query = require('query-string');
-_define('queryString', function() {
-    return query;
-});
-
-// flvjs
-const flvjs = require('flv.js/dist/flv').default;
-_define('flvjs', function() {
-    return flvjs;
-});
-
-// jstree
-const jstree = require('jstree');
-require('jstree/dist/themes/default/style.css');
```

Comment:
need to import this CSS somewhere - metadata editor is unstyled

Reply:
added import in a promise.all with jstree